.github/workflows/ci.yml

```diff
@@ -0,0 +1,87 @@
+name: CI
+
+on:
+  push:
+    branches: [master, staging]
+  pull_request:
+    branches: [master, staging]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  CHROME_BIN: /usr/bin/google-chrome
+
+permissions:
+  id-token: write
+  contents: write
+
+jobs:
+  test:
+    name: Checks & Build Test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 12
+      - name: Install Dependenices
+        run: npm install
+      - name: Run Test with CodeCov
+        env:
+          CHROME_BIN: /usr/bin/google-chrome
```

Comment:
shouldn't this automatically inherit the global `env` variable defined on line 14?

I don't care if we use the global or the step specific env, it just seems pointless to have both defined if the one paves over the other one in the only place we use it

Reply:
I'll remove one and try again